x-pack/plugins/security_solution_serverless/server/types.ts

```diff
@@ -60,5 +62,41 @@ export interface UsageMetrics {
 export interface UsageSource {
   id: string;
   instance_group_id: string;
-  instance_group_type: string;
+  instance_group_type?: string; // not seems part of step 1 fields https://github.com/elastic/mx-team/blob/main/teams/billing/services/usage_record_schema_v2.md
+}
+
+export interface SecurityUsageReportingTaskSetupContract {
+  core: CoreSetup;
+  logFactory: LoggerFactory;
+  taskManager: TaskManagerSetupContract;
+  cloudSetup: CloudSetup;
+  taskType: string;
+  taskTitle: string;
+  version: string;
+  meteringCallback: MeteringCallback;
+}
+
+export interface SecurityMetadataTaskStartContract {
```

Comment:
let's rename this to: `SecurityUsageReportingTaskStartContract`

Reply:
done:)